etc/apache2/vhosts.d/openqa.conf

```diff
@@ -0,0 +1,28 @@
+# sample apache config file for the openqa vhost. At the very
```

Comment:
Not sure where this file came from but this seems wrong

Reply:
Done, yes, it was a mistake